### PR TITLE
[zh] - docs/reference/command-line-tools-reference/kube-proxy.md

### PR DESCRIPTION
**Pull request by popomen**
ref: #29327

### CODE REVIEW

**Comment by tengqm** [comment]
@popomen Please submit your work in batch? The work item you signed on has other files to sync.

**Comment by popomen** [comment]
> @popomen Please submit your work in batch? The work item you signed on has other files to sync.

done~

File: content/zh/docs/reference/command-line-tools-reference/kube-apiserver.md [hunk 1]
```diff
@@ -1639,9 +1641,9 @@ Maximum number of seconds between log flushes
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-Sets the log format. Permitted formats: "json", "text".<br/>Non-default formats don't honor these flags: --add_dir_header, --alsologtostderr, --log_backtrace_at, --log_dir, --log_file, --log_file_max_size, --logtostderr, --one_output, --skip_headers, --skip_log_headers, --stderrthreshold, --vmodule, --log-flush-frequency.<br/>Non-default choices are currently alpha and subject to change without warning.
+Sets the log format. Permitted formats: "text".<br/>Non-default formats don't honor these flags: --add_dir_header, --alsologtostderr, --log_backtrace_at, --log_dir, --log_file, --log_file_max_size, --logtostderr, --one_output, --skip_headers, --skip_log_headers, --stderrthreshold, --vmodule, --log-flush-frequency.<br/>Non-default choices are currently alpha and subject to change without warning.
 -->
-设置日志格式。允许的格式："json"，"json"。<br/>
+设置日志格式。允许的格式："json"。<br/>
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-apiserver.md:1646]: json or text?

File: content/zh/docs/reference/command-line-tools-reference/kube-apiserver.md [hunk 2]
```diff
@@ -1690,12 +1692,11 @@ If non-zero, throttle each user connection to this number of bytes/sec. Currentl
...
 <!--
-The maximum number of mutating requests in flight at a given time. 
-When the server exceeds this, it rejects requests. Zero for no limit.
+This and --max-requests-inflight are summed to determine the server's total concurrency limit (which must be positive) if --enable-priority-and-fairness is true.
+Otherwise, this flag limits the maximum number of mutating requests in flight, or a zero value disables the limit completely.
 -->
-在给定时间内进行中变更类型请求的最大个数。
-当超过该值时，服务将拒绝所有请求。
-零表示无限制。
+如果 --enable-priority-and-fairness 为 true，那么该值和 --max-requests-inflight 的和将确定服务器的总并发限制（必须是正数）。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-apiserver.md:1698]:
```suggestion
如果 --enable-priority-and-fairness 为 true，那么此值和 --max-requests-inflight 的和将确定服务器的总并发限制（必须是正数）。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-apiserver.md [hunk 3]
```diff
@@ -1705,12 +1706,11 @@ When the server exceeds this, it rejects requests. Zero for no limit.
...
 <!--
-The maximum number of non-mutating requests in flight at a given time. 
-When the server exceeds this, it rejects requests. Zero for no limit.
+This and --max-mutating-requests-inflight are summed to determine the server's total concurrency limit (which must be positive) if --enable-priority-and-fairness is true.
+Otherwise, this flag limits the maximum number of non-mutating requests in flight, or a zero value disables the limit completely.
 -->
-在给定时间内进行中非变更类型请求的最大数量。
-当超过该值时，服务将拒绝所有请求。
-零表示无限制。
+如果 --enable-priority-and-fairness 为 true，那么该值和 --max-mutating-requests-inflight 的和将确定服务器的总并发限制（必须是正数）。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-apiserver.md:1712]:
```suggestion
如果 --enable-priority-and-fairness 为 true，那么此值和 --max-mutating-requests-inflight 的和将确定服务器的总并发限制（必须是正数）。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-apiserver.md [hunk 4]
```diff
@@ -2454,6 +2457,18 @@ If set, the file that will be used to secure the secure port of the API server v
...
 
+<tr>
+<td colspan="2">--tracing-config-file string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<!--
+File with apiserver tracing configuration.
+-->
+带有 apiserver 跟踪配置的文件。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-apiserver.md:2468]:
```suggestion
包含 API 服务器跟踪配置的文件。
```

File: content/zh/docs/reference/command-line-tools-reference/kube-controller-manager.md [hunk 5]
```diff
@@ -352,9 +352,10 @@ Filename containing a PEM-encoded X509 CA certificate used to issue cluster-scop
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!-- 
-The length of duration signed certificates will be given. 
+The max length of duration signed certificates will be given.
+Individual CSRs may request shorter certs by setting spec.expirationSeconds.
 -->
-所签名证书的有效期限。
+所签名证书的有效期限。个人的 CSR 可以通过设置 spec.expirationSeconds 来请求更短的证书。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/kube-controller-manager.md:358]:
```suggestion
所签名证书的有效期限。每个 CSR 可以通过设置 spec.expirationSeconds 来请求更短的证书。
```

**Comment by popomen** [comment]
Fixed~ Thanks for review~ @tengqm

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**